Sources/Plasma/FeatureLib/pfPython/cyAvatarGlue.cpp

```diff
@@ -639,6 +639,18 @@ PYTHON_METHOD_DEFINITION(ptAvatar, loadClothingFromFile, args)
     PYTHON_RETURN_BOOL(self->fThis->LoadClothingFromFile(filename));
 }
 
+PYTHON_METHOD_DEFINITION(ptAvatar, findBone, args)
+{
+    char* boneName = nullptr;
+    if (!PyArg_ParseTuple(args, "s", &boneName))
+    {
+        PyErr_SetString(PyExc_TypeError, "findBone expects a string bone name");
+        PYTHON_RETURN_ERROR;
+    }
+
+    return self->fThis->FindBone(ST::string::from_utf8(boneName));
```

Comment:
Can parse directly to a `ST::string`:

```suggestion
    ST::string boneName;
    if (!PyArg_ParseTuple(args, "O&|", PyUnicode_STStringConverter, &boneName))
    {
        PyErr_SetString(PyExc_TypeError, "findBone expects a string bone name");
        PYTHON_RETURN_ERROR;
    }

    return self->fThis->FindBone(boneName);
```